### ISSUE TITLE
Accessibility - color contrast missing for elements

### ISSUE THREAD

**Issue by MelSumner** [ember-learn/ember-styleguide#328]
## Expected Behavior
ember-styleguide to have correct color contrast 

## Actual Behavior
ember-styleguide is failing accessibility tests for color contrast

## Steps to Reproduce
check the ember-styleguide page using the axe-core plugin (below is my screenshot from doing the same):
![image](https://user-images.githubusercontent.com/4587451/84090163-847abe00-a9b6-11ea-8b00-d730b341badb.png)

**Comment by MinThaMie** [comment]
Will be fixed in https://github.com/empress/field-guide-default-template/issues/37
